apps/healthlocker/web/templates/layout/app.html.eex

```diff
@@ -29,6 +29,16 @@
 
     </script>
 
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-113820085-1"></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+
+      gtag('config', 'UA-113820085-1');
```

Comment:
@RobStallion this is _good_ but are we then pushing _more_ events to GTM datalayer...?